### ISSUE TITLE
Wrong frequency

### ISSUE THREAD

**Issue by MAGIKru** [xnk/pacific-tpms#3]
Official information about this FCC ID PAXPMV107J says that Frequency Range is 314.98 MHz, but not 315.98MHz as you mentioned in README.md. Using radio receiver I can confirm, that every 90 seconds on this frequency I can see short signal.

[https://fccid.io/PAXPMV107J](https://fccid.io/PAXPMV107J)

**Comment by xnk** [comment]
Yep, definitely a typo of mine, fixed the README! The receiver code centers its window around 314.985MHz and the transmitter code center frequency is 314.98MHz.
